### ISSUE TITLE
Default addons should be specified by name

### ISSUE THREAD

**Issue by amitkrout** [minishift/minishift#1355]
No defaults addons are specified in the doc, [minishift install addons](https://docs.openshift.org/latest/minishift/command-ref/minishift_addons_install.html) section.

IMO it should be something like 

```
--defaults   If true, installs all Minishift default add-ons (default : anyuid, admin-user, xpaas, registry-route)
```

**Comment by anjannath** [comment]
@amitkrout are the command refs auto generated?

**Comment by amitkrout** [comment]
@anjannath No it's not. You need to add that. I cant find the file from master you mentioned above.

**Comment by anjannath** [comment]
@amitkrout I think that file is created after building the docs.

**Comment by budhrg** [comment]
@anjannath , all cmd refs are created with https://github.com/minishift/minishift/blob/master/Makefile#L123. 

So, any changes you need, it should be done to respective command go file. Here https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/addon/addon_install.go.

**Comment by gbraad** [comment]
Fixed with 99a16d03da3489c0ea021762f0fb1ee4d6f524ec
